test/index.test.js

```diff
@@ -43,7 +42,7 @@ describe('fixtures', () => {
 
 			await sleep(1);
 
-			const output = await buildDirectory(fixtureDir);
```

Comment:
Unfortunately we need to drop this aspect of the suite. gzip & brotli sizes aren't guaranteed to be stable, not through Node versions or on different systems. As such, I was seeing tests pass locally just fine that would then be 1-2b off on the CI.

Besides, all that this aspect was testing was that the gzip & brotli size estimates were stable, which isn't particularly important. If the expected output is stable, it doesn't really matter to us what the gzip/brotli size ends up being.